Client/Frontend/Widgets/TabsButton.swift

```diff
@@ -118,6 +113,7 @@ extension TabsButton: Themeable {
         
         textColor = top ? theme.colors.tints.header : theme.colors.tints.footer
```

Comment:
Previously setting `textColor` called `updateButtonVisuals()` which causes the crash because `highlightTextColor` was not set yet